<launch>
    <arg name="robot_ns" default="robot0" />
    <node name="stdr_publisher_node" type="stdr_publisher_node" pkg="robot_no_crash" ns="$(arg robot_ns)" output="screen" />
    <!-- <node name="stdr_subscriber_node" type="stdr_subscriber_node" pkg="robot_no_crash" ns="$(arg robot_ns)" /> -->
    <remap from="laser_1" to="laser_0" />
</launch>